app/presenters/publishing_api/embassies_index_presenter.rb

```diff
@@ -8,21 +8,23 @@ def content_id
     end
 
     def content
-      content = BaseItemPresenter.new(
-        nil,
-        title:,
-        update_type: "minor",
-      ).base_attributes
+      @content ||= begin
+        content = BaseItemPresenter.new(
+          nil,
+          title:,
+          update_type: "minor",
+        ).base_attributes
 
-      content.merge!(
-        base_path:,
-        details:,
-        document_type: "embassies_index",
-        rendering_app: Whitehall::RenderingApp::COLLECTIONS_FRONTEND,
-        schema_name: "embassies_index",
-      )
+        content.merge!(
+          base_path:,
+          details:,
+          document_type: "embassies_index",
+          rendering_app: Whitehall::RenderingApp::COLLECTIONS_FRONTEND,
+          schema_name: "embassies_index",
+        )
 
-      content.merge!(PayloadBuilder::Routes.for(base_path))
+        content.merge!(PayloadBuilder::Routes.for(base_path))
+      end
     end
```

Comment:
Is it worth adding a test to say that `BaseItemPresenter.new` is only called once when calling `EmbassiesIndexPresenter.new.content` two or more times?

Reply:
Good suggestion. Have just updated this.